odbc/odbc.md

```diff
@@ -0,0 +1,287 @@
+# **ODBC**
+
+Open Database Connectivity (ODBC) is a standardized API for connecting to databases that is cross-platform and cross-DBMS (Database Management System). By installing an general ODBC driver manager and a driver for your specific DBMS, the code that interacts with the ODBC API can be generalized. This makes ODBC a very powerful tool for both development and for production, as the application can be deployed on any system and talk to any DBMS as long as it has the correct drivers installed.
+
+## **Table of Contents**
+
+* [ODBC](#odbc)
+  * [Table of Contents](#table-of-contents)
+  * [Why ODBC?](#why-odbc)
+* [Installation](#installation)
+  * [Installation on IBM i](#installation-on-ibm-i)
+    * [Driver Manager](#driver-manager-0)
+    * [Driver](#driver-0)
```

Comment:
The above two links don't work when I preview it.

Reply:
first instance is #driver-manager, not #driver-manager-0 apparently. Fixed